webapp/channels/src/components/suggestion/search_channel_with_permissions_provider.tsx

```diff
@@ -235,4 +243,4 @@ export default class SearchChannelWithPermissionsProvider extends Provider {
             component: SearchChannelWithPermissionsSuggestion,
         });
     }
-}
+}
```

Comment:
style checker will complain because there is no new line

Reply:
done